package.json

```diff
@@ -5,7 +5,7 @@
   "main": "main.js",
   "scripts": {
     "test": "jest",
-    "fmt": "prettier --write '**/*.{js,json,md}'"
```

Comment:
I think it's better to specify markdown( `md` ) as the target extension for prettier. This is because it will automatically correct formatting errors in markdown.

If you think it's a bad idea to use markdown, could you tell me a reason?

Reply:
Hi @shinshin86 
thanks for asking, for some reason, with 'md' on **fmt** script, prettier does not format markdown also throw some errors

when I removed 'md' from **fmt** script, prettier formats markdown file in right way.

may be there is something wrong in my computer. idk

Reply:
@frani Thanks for the reply.

Oh I see.
I think prettier for markdown is useful to have, but I don't think it's essential.
So I've decided to apply your changes.
This PR will merge.

Thank you!

Reply:
thanks to you to replay again, and thanks for be compressive 👍